papers/tesseroid-variable-density.md

```diff
@@ -1,20 +1,27 @@
 ---
 title: "Gravitational field calculation in spherical coordinates using variable densities in depth "
-date: 2019-07-01
+date: 2019-06-05
```

Comment:
This is the acceptance date. Should I change it to the publication date (11 June 2019)?

Reply:
Yes we should

Reply:
Done! :+1: